workflows/iuf/operations/vcs-upload/vcs-upload-content.yaml

```
@@ -74,6 +74,11 @@ spec:
                   value: "{{=jsonpath(inputs.parameters.global_params, '$.stage_params.process-media.current_product.parent_directory')}}/{{=jsonpath(inputs.parameters.global_params, '$.product_manifest.current_product.manifest.content.vcs.path')}}"
                 - name: cf_import_gitea_org
                   value: cray
+                - name: cf_import_gitea_repo
+                  value: >
+                    "repo_name" in jsonpath(inputs.parameters.global_params, "$.product_manifest.current_product.manifest.vcs") ?
+                    jsonpath(inputs.parameters.global_params, "$.product_manifest.current_product.manifest.vcs.repo_name") :
+                    ""
```

Curly braces here too:

```suggestion
                    {{="repo_name" in jsonpath(inputs.parameters.global_params, "$.product_manifest.current_product.manifest.content.vcs") ?
                    jsonpath(inputs.parameters.global_params, "$.product_manifest.current_product.manifest.content.vcs.repo_name") :
                    ""}}
```